utopia-remix/app/models/project.server.ts

```diff
@@ -10,6 +10,7 @@ const selectProjectWithoutContent: Record<keyof ProjectWithoutContent, true> = {
   modified_at: true,
   deleted: true,
   ProjectAccess: true,
+  ProjectAccessRequest: true,
```

Comment:
nope, we should not include this by default. we should only include it when it makes sense to have it.

Reply:
[`c3ee65a` (#5033)](https://github.com/concrete-utopia/utopia/pull/5033/commits/c3ee65a835fda7c2e0743b2f77551e4317448152) fixes this

Reply:
fixed (by you :) )